test/acl.test.js

```diff
@@ -5,10 +5,13 @@
 
 'use strict';
 var assert = require('assert');
+var expect = require('./helpers/expect');
```

Comment:
reviewed some of your own new tests: should we use expect from loopback helpers or vanilla chai expect?